Framework/AnalysisSupport/src/AODJAlienReaderHelpers.cxx

```diff
@@ -147,6 +145,31 @@ static inline auto extractOriginalsTuple(framework::pack<Os...>, ProcessingConte
   return std::make_tuple(extractTypedOriginal<Os>(pc)...);
 }
 
+void AODJAlienReaderHelpers::dumpFileMetrics(Monitoring& monitoring, TFile* currentFile, int tfPerFile, int tfRead)
+{
+  std::string monitoringInfo("lfn=");
```

Comment:
You can probably use fmt::format() for this, which would end up more compact.

Reply:
Done